src/primitives/float32.mli

```diff
@@ -62,6 +62,8 @@ val ge : t -> t -> bool
 
 val of_string : string -> t
 
+val of_string_opt : string -> t option
+
```

Comment:
While you're at it, could you rename `of_string` to `of_string_exn` and rename `of_string_opt` to `of_string` ? You'll also have to change it in `float64`, `int32` and `int64` but I expect it to be quite straightforward.

Reply:
Ok, it's fixed